common/src/main/java/org/apache/uniffle/common/util/ByteBufUtils.java

```diff
@@ -96,4 +96,11 @@ public static ByteBuf byteStringToByteBuf(ByteString bytes) {
     final ByteBuffer byteBuffer = bytes.asReadOnlyByteBuffer();
     return Unpooled.wrappedBuffer(byteBuffer);
   }
+
+  public static ByteBuf createByteBufFrom(ByteBuf from) {
```

Comment:
How about renaming this method to `from` ?

Reply:
`copy`?